README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href='https://www.omise.co'><img src='https://assets.omise.co/assets/omise-logo-ed530feda8c7bf8b0c990d5e4cf8080a0f23d406fa4049a523ae715252d0dc54.svg' height='60'></a></p>
+<p align="center"><a href='https://www.omise.co'><img src='https://user-images.githubusercontent.com/10651523/41644923-b79aaf74-7499-11e8-9c7e-85186421c841.png' height='60'></a></p>
```

Comment:
@jacekstan cool thanks for fixing this link!!

Btw, can you replace this url with https://cloud.githubusercontent.com/assets/2154669/26388730/437207e4-4080-11e7-9955-2cd36bb3120f.png instead? 🙇 🙇